neural_network/dropout.py

```diff
@@ -0,0 +1,293 @@
+import numpy
+
+numpy.seterr(all='ignore')
+
+
+'''
+activation function
+'''
+def sigmoid(x):
+    return 1. / (1 + numpy.exp(-x))
+
+def dsigmoid(x):
+    return x * (1. - x)
+
+def tanh(x):
+    return numpy.tanh(x)
+
+def dtanh(x):
+    return 1. - x * x
+
+def softmax(x):
+    e = numpy.exp(x - numpy.max(x))  # prevent overflow
+    if e.ndim == 1:
+        return e / numpy.sum(e, axis=0)
+    else:
+        return e / numpy.array([numpy.sum(e, axis=1)]).T  # ndim = 2
+
+def ReLU(x):
+    return x * (x > 0)
+
+def dReLU(x):
+    return 1. * (x > 0)
+
+
+
+'''
+Dropout
+'''
+class Dropout(object):
+    def __init__(self, input, label,\
```

Comment:
Please remove all six backslashes from this file.  They are not needed and their use is discouraged in PEP8.

Reply:
okay